sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -59,8 +60,9 @@
     private String testName;
     private final Scheduler scheduler = Schedulers.parallel();
 
-    private static final byte[] CONTENTS_BYTES = "Some-contents".getBytes(StandardCharsets.UTF_8);
+    protected static final byte[] CONTENTS_BYTES = "Some-contents".getBytes(StandardCharsets.UTF_8);
```

Comment:
Is there a reason to make this protected?

Reply:
I am using this in child classes.